audit/src/main/java/org/wildfly/security/audit/PeriodicRotatingFileAuditEndpoint.java

```diff
@@ -174,6 +174,10 @@ public static class Builder extends FileAuditEndpoint.Builder {
             super();
         }
 
+        private static <T extends Comparable<? super T>> T min(T a, T b) {
```

Comment:
@gerjantd Just a total minor, can  you please place this method below the public methods in the Builder class for convention reasons? Thank you!

Reply:
@Skyllarr  done. I squashed the commits. Do you need a new PR or can you take it from here?